app/models/invoice_subscription.rb

```diff
@@ -14,15 +14,13 @@ class InvoiceSubscription < ApplicationRecord
   monetize :total_amount_cents, disable_validation: true, allow_nil: true
 
   scope :order_by_charges_to_datetime,
-        lambda {
-          condition = <<-SQL
-            COALESCE(
-              (invoice_subscriptions.properties->>\'to_datetime\')::timestamp, invoice_subscriptions.created_at
-            ) DESC
-          SQL
+    lambda {
```

Comment:
Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line. (https://rubystyle.guide#no-double-indent)